src/screens/CreateWalletScreen.tsx

```diff
@@ -71,25 +71,28 @@ export class CreateWalletScreen extends React.PureComponent<Props, State> {
 
   navigateToImportWallet = () => this.props.navigation.navigate(Route.ImportWallet);
 
+  getWalletClassByIndex = (index: number) => {
```

Comment:
why not choose from an array of WalletTypes?

Reply:
for me it is quite readable like this,  we can easily see which number is which wallet and which one is the default one, without counting the position of wallet in array